<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeDialog</class>
 <widget class="QDialog" name="EmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>271</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>298</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QComboBox" name="activeCombobox"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="activeLabel">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="idText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="roleCombobox"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="surnameText"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="companyLabel">
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameText"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="surnameLabel">
     <property name="text">
      <string>Surname</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="passwordText">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QTextEdit" name="noteText">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="roleLabel">
     <property name="text">
      <string>Role</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="noteLabel">
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="usernameText"/>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="companyCombobox"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="sysroleCombobox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sysroleLabel">
     <property name="text">
      <string>System Role</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameText</tabstop>
  <tabstop>surnameText</tabstop>
  <tabstop>usernameText</tabstop>
  <tabstop>passwordText</tabstop>
  <tabstop>roleCombobox</tabstop>
  <tabstop>companyCombobox</tabstop>
  <tabstop>sysroleCombobox</tabstop>
  <tabstop>noteText</tabstop>
  <tabstop>activeCombobox</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>idText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
